### ISSUE TITLE
Grishna Raven does not remove Polymorph effect when in Bestial Wrath

### ISSUE THREAD

**Issue by netherspite** [ccshiro/cc-buglist#1494]
![wowscrnshot_082315_184241](https://cloud.githubusercontent.com/assets/5777848/9429809/f8404606-49e5-11e5-9db7-ad23b1f59ca3.jpg)

**Comment by Chulain** [comment]
http://www.wowhead.com/npc=20327/grishna-raven
on 2010/01/06 (Patch 3.3.0)
Whilst under Bestial Wrath these are immune to Hibernate, and Bestial Wrath being cast removes the Hibernate effect.

**Comment by netherspite** [comment]
Yes, I read this comment on wowhead and I guess the same applies to Polymorph effect.
